src/classes/Integration.cls

```diff
@@ -1,21 +1,22 @@
 global class Integration{
 
-    public String getUsersFromS3(){
-        HTTPResponse resp = AWSUtility.getS3file ('sfdc-training', 'FinancialAdvisors.json');
-        return resp.getBody();
-    }
-
-    public void upsertUsers(String json){
+    /**
+    * @description This method takes all users retreived from the aws s3 bucket and upserts them in  
+    *              Salesforce. In case there is a user in Salesforce who's no longer in the json
+    *              it is deleted
+    *
+    * @param resp HttpResponse obtained from the callout to de aws s3 bucket, to get the file
+    */
+    public static void upsertUsers(HttpResponse resp){
```

Comment:
While it's not necessarily a bad practice, there's no need to pass in the HttpResponse type, it'd be clearer if this method received either the JSON string or the parse list of Financial Advisors.